### ISSUE TITLE
Get access to hotjar control and monitoring panels

### ISSUE THREAD

**Issue by delapuente** [Qiskit/qiskit.org#295]
Part of #292

**Comment by marinaaisa** [comment]
I managed to get access to Hotjar but I couldn't create a new site since it's a feature only available for admins. So I opened a new issue on their backlog to request them to create a new site on Hotjar for qiskit.org and community.qiskit.org: https://github.ibm.com/IBM-Q-Software/om-digital-planning/issues/324

**Comment by marinaaisa** [comment]
Since I was away, I couldn't review this issue until now and it seems that both Christophers didn't understand me well: https://github.ibm.com/IBM-Q-Software/om-digital-planning/issues/324 I have asked them again for creating a new site for qiskit.org in IBM's Hotjar.

**Comment by marinaaisa** [comment]
Christopher Fisher replied me in this issue: https://github.ibm.com/IBM-Q-Software/om-digital-planning/issues/324 saying that we should contact Gargi Sharma and Kholo Estipona . @lerongil open a direct message on Slack with the 4 of us and they open a ticket for the new site. We are waiting on the resolution of this ticket.

**Comment by marinaaisa** [comment]
This is the reply about creating a new site on Hotjar:
![Screen Shot 2019-11-25 at 9 49 18 AM](https://user-images.githubusercontent.com/8567677/69525667-e61e9200-0f68-11ea-909a-ae7730ae7476.png)
you can find it [here](https://nasales.ibmus2.ibm.com/helpdesk/HDW_Prod.nsf/newTicketSupport.xsp?databaseName=D25DBW01!!helpdesk/HDD_Prod.nsf&documentId=0D83F5FABAAF0932852584B1006BA31F&action=openDocument).

It seems that we will have to wait a bit longer than expected...

**Comment by marinaaisa** [comment]
## We finally got a site for Qiskit.org on Hotjar! 🥳
<img width="396" alt="Screen Shot 2019-11-25 at 4 10 34 PM" src="https://user-images.githubusercontent.com/8567677/69553339-55fc3f00-0fa0-11ea-864f-d471d71732c3.png">


You can find it clicking on the building icon on the left part of the page, on Hotjar, and then a selector appears where you can select Qiskit.org

<img width="462" alt="Screen Shot 2019-11-25 at 4 24 58 PM" src="https://user-images.githubusercontent.com/8567677/69553471-89d76480-0fa0-11ea-9c4e-a21b74878868.png">

**Comment by delapuente** [comment]
What's the status of this @marinaaisa, @lerongil ? I think someone mentioned issues in recorded sessions, is it still a problem?

**Comment by lerongil** [comment]
I believe this should be closed. @marinaaisa thoughts?

**Comment by delapuente** [comment]
Perr offline conversation with @marinaaisa this can be closed.
